tests/adapters_t/sites_t/test_slurm.py

```diff
@@ -14,11 +14,12 @@
 from datetime import datetime, timedelta
 
 import asyncio
+import logging
 
 __all__ = ["TestSlurmAdapter"]
 
 TEST_RESOURCE_STATUS_RESPONSE = """
-1390065|None assigned|PENDING
+1390065||PENDING
```

Comment:
Can *None assigned* not appear anymore?

Reply:
No,`None assigned` only appears when using `sacct`. Using `squeue` the host name is empty, when no worker node has been assigned yet.